src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -181,27 +203,47 @@ def get_paths_to_platform_and_hwsku_dirs():
 
     return (platform_path, hwsku_path)
 
-
-def get_path_to_port_config_file():
+def get_path_to_port_config_file(hwsku=None, asic=None):
     """
     Retrieves the path to the device's port configuration file
```

Comment:
Enhance the docstring to describe the Args. Mention that we allow passing `hwsku` because when loading the initial configuration on the device, the HwSKU is not yet present in ConfigDB.

`asic` should be passed on multi-ASIC devices only, it should be omitted on single-ASIC platforms.

Reply:
fixed

Reply:
Can you format the Args like this: https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/chassis_base.py#L148?